docs/src/pages/components/geo/fragments/marker-with-popup.react.mdx

```diff
@@ -0,0 +1,50 @@
+```jsx
+import { AmplifyMap } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+import React from 'react';
+import { Marker, Popup } from 'react-map-gl';
+
+import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+
+Amplify.configure(awsExports);
+
+function MarkerWithPopup({ latitude, longitude }) {
+  const [showPopup, setShowPopup] = React.useState(false);
+
+  const handleMarkerClick = ({ originalEvent }) => {
+    originalEvent.stopPropagation();
+    setShowPopup(true);
+  };
+
+  return (
+    <>
+      <Marker
+        latitude={latitude}
+        longitude={longitude}
+        onClick={handleMarkerClick}
+      />
+      {showPopup && (
+        <Popup
+          latitude={latitude}
+          longitude={longitude}
+          offset={{ bottom: [0, -40] }}
+          onClose={() => setShowPopup(false)}
+        >
+          <h2>Marker Information</h2>
+          Some information about a location.
```

Comment:
Perhaps worth demonstrating how to use the `Heading` and `Text` components from `@aws-amplify/ui-react`?

Reply:
OK, I'll update that!